Scripts/Mend SCA/get-malicious-packages.sh

```diff
@@ -1,25 +1,25 @@
 #!/bin/bash
 # Prerequisites:
-# apt install jq curl xargs 
+# apt install jq curl
 # WS_USERKEY - An administrator's userkey
 # WS_EMAIL - The administrator's email
 # WS_APIKEY - API Key for organization
```

Comment:
APIKEY should be optional as it is only required when a user has access to multiple orgs.  Service users with access to a single org are a common practices for calling apis instead of regular users